spring-cloud-skipper-server-core/src/test/resources/repositories/binaries/test/index.yml

```diff
@@ -94,3 +94,39 @@ packageHomeUrl: http://cloud.spring.io/spring-cloud-stream-app-starters/
 tags: helloworld
 maintainer: https://github.com/markpollack
 description: The hello world app says hello.
+
+---
+apiVersion: skipper.spring.io/v1
+kind: SkipperPackageMetadata
+origin: samples-package-repository
+name: testapp
+version: 1.0.0
+packageSourceUrl: https://github.com/spring-cloud/spring-cloud-deployer
+packageHomeUrl: https://cloud.spring.io/spring-cloud-dataflow/
+tags: logging, sink
+maintainer: https://github.com/jvalkeal
+description: The testapp up starts and conditionally fails.
```

Comment:
I guess this doesn't fail. will update the description when merging.

Reply:
Yeah, those are just bogus field values. Or afaik not actually use anywhere.